student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
+
```

Comment:
- [ ] As I mentioned in the previous comment, kindly make sure you use the same approach, you should update the constructor method to have the same behavior like the one of the `Person` class. `name` should have default value of 'Unknown', 'parent_permission' of true, and others. You can use the same example like in the previous comment. Just change `specialization` to `classroom`